### ISSUE TITLE
Plot for Epochs, Accuracy and Avg Loss

### ISSUE THREAD

**Issue by Viddesh1** [konrad-gajdus/miniMNIST-c#2]
Hello @konrad-gajdus  👋 ,

Thanks for the implementation of minimal neural network in C programming language for MNIST 0-9 digits classification.

I was playing with the code in my local machine. I came to understanding that it takes a significant amount of the time reaching 20 epochs. I think it will be great to plot and visualize the epochs, accuracy and average loss metrics of the trained model.

Regards!
Viddesh
